src/pkg/packaging/deb/package.targets

```diff
@@ -283,6 +284,88 @@
           UseHardlinksIfPossible="False" />
 
   </Target>
+  
+  <Target Name="GenerateRuntimeMetaDeb">
+  
+    <PropertyGroup>
+      <DebPackageName>$(RuntimeMetaDebPkgName)</DebPackageName>
+      <DebPackageVersion>$(SharedFrameworkNugetVersion)</DebPackageVersion>
+      <DebFile>$(DotnetRuntimeMetaPackageInstallerFile)</DebFile>
+      <ConfigJsonName>dotnet-runtime-deps-debian_config.json</ConfigJsonName>
+      <ConfigJsonFile>$(debPackaginfConfigPath)$(ConfigJsonName)</ConfigJsonFile>
+      <debIntermediatesDir>$(PackagesIntermediateDir)$(DebPackageName)/$(DebPackageVersion)</debIntermediatesDir>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <debLayoutDirectory>$(debIntermediatesDir)/debianLayoutDirectory/</debLayoutDirectory>
+      <debLayoutAbsolute>$(debLayoutDirectory)$</debLayoutAbsolute>
+      <debLayoutPackageRoot>$(debLayoutDirectory)package_root</debLayoutPackageRoot>
+      <debLayoutSamples>$(debLayoutDirectory)samples</debLayoutSamples>
+      <debLayoutDocs>$(debLayoutDirectory)docs</debLayoutDocs>
+    </PropertyGroup>
+
+    <RemoveDir Condition="Exists('$(debIntermediatesDir)')" Directories="$(debIntermediatesDir)" />
+    <MakeDir Directories="$(debIntermediatesDir)" />
+
+    <!-- Compute ICU Version --> 
+    <Exec Command="apt-cache search --names-only '^libicu[0-9]+$' | cut -d' ' -f1" 
```

Comment:
This appears to be duplicated below.  Can you refactor it to eliminate the duplication?

Reply:
will take care of this as part of #3325

Reply:
It feels wrong to add duplication now, and plan on refactoring it later as part of a proposal that we aren't sure we are going to do.  I think this duplication shouldn't be that hard to fix now.